e2e/specs/st_camera_input.spec.js

```diff
@@ -44,9 +44,9 @@ describe("st.camera_input", () => {
       .wait(1000)
       .should("not.be.disabled")
       .contains("Take Photo")
-      .click();
+      .click({ force: true });
 
-    cy.get("img").should("have.length.at.least", 2);
+    cy.get("img", { timeout }).should("have.length.at.least", 2);
 
     cy.get("[data-testid='stImage']", { timeout }).should("have.length.at.least", 1);
   });
```

Comment:
Think it would be worthwhile to add auto-retry to this test as well with: `{ retries: { runMode: 1 } },` like in the test below

Reply:
Done!